packages/@webex/internal-plugin-metrics/src/call-diagnostic/config.ts

```diff
@@ -4,6 +4,8 @@
 
 import {ClientEventError} from '../metrics.types';
 
+export const CALL_DIAGNOSTIC_LOG_IDENTIFIER = 'call-diagnostic-events -> ';
```

Comment:
is this arrow following a convention/pattern for log identifier in the sdk?

Reply:
not really, i don't think there is a convention